docs/maintenance/Contributor_Tiers.mdx

```diff
@@ -52,15 +52,16 @@ These tiers of contributors, committers, and maintainers roughly reflect how we'
       <th>Maintainer</th>
       <td>
         ~10 points in issues
-        <br />
+
         ~10 points in pull requests
-        <br />
+
         ~30 points total
         <br />3 of 6 active months
       </td>
       <td>~20 points</td>
       <td>~2 shares / active month</td>
     </tr>
+
```

Comment:
Nit: unrelated?

```suggestion
```

Reply:
this space is evoke with format script. and i fix this, the format ci is failed.